packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -1200,6 +1312,32 @@ class _LinearProgressIndicatorDefaultsM2 extends ProgressIndicatorThemeData {
   double get linearMinHeight => 4.0;
 }
 
+class _CircularProgressIndicatorDefaultsM3Year2023 extends ProgressIndicatorThemeData {
+  _CircularProgressIndicatorDefaultsM3Year2023(this.context, { required this.indeterminate });
+
+  final BuildContext context;
+  late final ColorScheme _colors = Theme.of(context).colorScheme;
+  late final bool indeterminate;
+
+  @override
+  Color get color => _colors.primary;
+
+  @override
+  double get strokeWidth => 4.0;
+
+  @override
+  double? get strokeAlign => CircularProgressIndicator.strokeAlignCenter;
+
+  @override
+  BoxConstraints get constraints => const BoxConstraints(
+    minWidth: 36.0,
```

Comment:
Earlier i tried 48x48 and found it to be too big so i compared it with Android components, it is 36 pixels with padding comes to 48x48. The actual shape is still 36x36. I can file an issue for incorrect specs.

Reply:
Filed an issue https://github.com/flutter/flutter/issues/158162

Reply:
Ah got it. I just asked the Material team to see if they have any suggestions. Probably it's just Android team hasn't updated this component since I just feel the Material Design website should be the only source of truth:)